app/controllers/ministers_controller.rb

```diff
@@ -2,9 +2,17 @@ class MinistersController < ApplicationController
   around_action :switch_locale
 
   def index
+    ab_test = GovukAbTesting::AbTest.new(
+      "GraphQLMinistersIndex",
+      allowed_variants: %w[ContentStore GraphQL],
+      control_variant: "ContentStore",
```

Comment:
I think it's good practice to have three variants A,B,Z where Z is excluded from the test, and A and B are always equal in weight. 

So in this case A and Z might be both be set to ContentStore. 

Then in Fastly you can set 100% to Z to confirm that the cookie is being set, and all the data is coming through. And then scale up keeping A and B equal, so instead of https://github.com/alphagov/govuk-fastly/pull/144/files you probably would have 98% Z, 1%A and 1%B

Reply:
See https://github.com/alphagov/govuk-fastly/pulls?q=is%3Apr+is%3Aclosed+vertex+ for prior art

Reply:
Ooh, that's a great approach.  I've updated to have three variants: A, B and Z.  A and Z will use Content Store, and B will use GraphQL.  I'll update the Fastly config to enable to tests and point 100% of traffic to variant Z for now.